mypy/fastparse2.py

```diff
@@ -599,6 +599,7 @@ def visit_ClassDef(self, n: ast27.ClassDef) -> ClassDef:
         cdef.line = n.lineno + len(n.decorator_list)
         cdef.column = n.col_offset
         cdef.end_line = n.lineno
+        cdef.end_column = n.col_offset
```

Comment:
Shouldn't this be None for consistency with `fastparse.py`?

Reply:
Yes, you're right.